tests/Refinery/String/StripTagsTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2017 Stefan Hecken <stefan.hecken@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+use ILIAS\Refinery;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * TestCase for SplitString transformations
+ *
+ * @author Stefan Hecken <stefan.hecken@concepts-and-training.de>
+ */
+class StripTagsTest extends TestCase {
+	const STRING_TO_STRIP = "I <script>contain</a> tags.";
+	const EXPECTED_RESULT = "I contain tags.";
+
+	protected function setUp() : void	{
+		$this->f = new \ILIAS\Refinery\Factory(
+			$this->createMock(\ILIAS\Data\Factory::class),
+			$language = $this->createMock('\ilLanguage')
+		);
+		$this->strip_tags = $this->f->string()->stripTags();
+	}
+
+	public function testTransform() {
+		$res = $this->strip_tags->transform(self::STRING_TO_STRIP);
+		$this->assertEquals(self::EXPECTED_RESULT, $res);
+	}
```

Comment:
Maybe adding a test that throws the exception would be nice. Just to know that the intention is clear, that we want to work with strings here,

@mjansenDatabay  @klees  I have an additional question here too: Do you mind to add tests for both: `transform` and `applyTo`? Or do you think one test (either `transfrom` or `applyTo`) is enough?

Reply:
IMHO one test, either for `transform` or `applyTo`, is sufficient. I highly appreciate a test for the `unhappy path`.

Reply:
Hi @legionth, @mjansenDatabay!

I added the test for the sad path.

About the relation between `applyTo` and `transform`: Basically both should do the same. So if both are implemented this would be a bad sign for me. I would expect every `Transformation` to implement only one of these and derive the other via a trait. Maybe we should put this more clearly somewhere in the docu of `Refinery\Transformation`.

Thanks for looking into this!

Reply:
@klees 

> So if both are implemented this would be a bad sign for me

+1, indeed!

> Maybe we should put this more clearly somewhere in the docu of Refinery\Transformation.

@legionth Could you please make a PR for this README addition? Thx in advance.

Reply:
Sure